### ISSUE TITLE
Enhancement, And Issue.

### ISSUE THREAD

**Issue by LuckyEyes1** [MareMare/DummyFileCreator#79]
A dummy file creator. Could use this, to make custom filetypes. ._. Um, my question is what should the buffer size be?
My file or console seems to reset the timer at the default on the console. Is it supposed to restart while the file is building? It gets about to the 6% then it stops and resets. I don't know if this console is looking for something online, or the bytes based on processing speed. Um, yeah. Just a thought. Though the file wanted is a big file from scratch.

**Comment by MareMare** [comment]
Hi @LuckyEyes1, Thank you for your question.

> Um, my question is what should the buffer size be?

The command line option `-b` or `--buffer` specifies the buffer size, but it does not work correctly. I will fix this later.

> My file or console seems to reset the timer at the default on the console. Is it supposed to restart while the file is building? It gets about to the 6% then it stops and resets.
> I don't know if this console is looking for something online, or the bytes based on processing speed. Um, yeah. Just a thought.

Oops, that's another bug. I'll fix it as well.
In my environment, if the file size is 1GB, the file can be generated without resetting the progress, but not if the file size is 5GB.

![Animation](https://github.com/user-attachments/assets/285d293e-d32f-468a-b21a-ba8257b226dd)

**Comment by MareMare** [comment]
Hi, @LuckyEyes1 
A modified version has been released as [v2.0.1](https://github.com/MareMare/DummyFileCreator/releases/tag/v2.0.1).

It should now work without resetting progress. Also, the buffer size can be changed by specifying the `-b 4MB` or `--buffer 4MB` command line option.

An example is shown below:
```ps1
.\DummyFileCreator.Console.exe -f dummy.txt -z -b 4MB -s 5GB
```

<details><summary>Windows Terminal:</summary>

![Animation](https://github.com/user-attachments/assets/c8fc4ee7-efd6-4da0-80cb-6737622b55e7)

</details>
